Riot/Modules/TabBar/MasterTabBarController.m

```diff
@@ -847,6 +864,27 @@ - (void)setupLeftBarButtonItem
     }
 }
 
+-(void)setupTitleView
+{
+    titleLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 0, 0)];
```

Comment:
We can do it later in another PR, but it can be interesting to make this view in a dedicated Swift class conforming to `Themable`. As MasterTabBarController will disappear in the future.